components/divider/index.css

```diff
@@ -10,46 +10,110 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-divider.css";
+/* divider/index.css
+ *
```

Comment:
Interesting comment block - will there be more added here? :) (nit)

Reply:
Yeah that's a placeholder for documentation that we don't need, I agree it should be removed. I realize I left a similar snippet in progress circle, so I removed that also.